### DISCUSSION TITLE
Value '*' for key 'com.apple.developer.associated-domains' is not supported

### DISCUSSION THREAD

**Discussion by DeathStalker403** [fastlane/fastlane#17025] [Q&A]
when trying to upload a build to testflight we are getting the  following 

`Invalid Code Signing Entitlements. Your application bundle's signature contains code signing entitlements that are not supported on iOS. Specifically, value '*' for key 'com.apple.developer.associated-domains' is not supported.`

Any idea on how to get rid of the above error. We are using resign to sign the ipa as gym is unable to discover the profiles.

Fastlane version 2.149.1

**Comment by max-ott** [comment]
Please update to the latest `fastlane` version and correct your entitlements file. As the error suggests `*` is an invalid value. If you are only resigning the IPA, let the developer know to correct the error.

**Reply by jambstud** [reply]
@max-ott   what kind of error do you think could be   that developer should be pointed ?
We are blocked on same kind of situation

**Comment by jambstud** [comment]
fastlane sigh resign  --verbose  gym/APP.ipa  -i  “Apple Distribution: . (Rere123)” -p “file.mobileprovision"   use_app_entitlements 
not updating it with a custom value

Any recomandation to import / combine it with a custm one : file1.entitlements

**Comment by jambstud** [comment]
in our case the *  is generated by   the command:
fastlane sigh --team_id SOMEID  -a PROFILE  -q "file.mobileprovision"

file.mobileprovision contains
				<key>com.apple.developer.associated-domains</key>
		<string>*</string>

Not sure how we can change his value
